generators/config/templates/drivers/poltergeist_driver.rb

```diff
@@ -0,0 +1,12 @@
+# :poltergeist driver
+
+CapybaraHelpers.load_driver_gem!(:poltergeist, 'capybara/poltergeist', 'poltergeist')
+
+Capybara.register_driver :poltergeist do |app|
+  Capybara::Poltergeist::Driver.new(
+    app,
+    js_errors: !Howitzer.phantom_ignore_js_errors,
+    phantomjs_options: ["--ignore-ssl-errors=#{Howitzer.phantom_ignore_ssl_errors ? 'yes' : 'no'}"]
+  )
+end
+Capybara.current_session.driver.add_headers('User-Agent' => Howitzer.user_agent)
```

Comment:
it should be somewhere between line 5 and 11, because in other case on testing API where browser is not required, we will have opened browser